<ui version="4.0" >
 <class>StrategiaProve</class>
 <widget class="QDialog" name="StrategiaProve" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Strategia Prove</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="contradaLabel" >
       <property name="text" >
        <string>textLabel2</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="montaLabel" >
       <property name="text" >
        <string>textLabel2</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QVBoxLayout" />
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="Q3GroupBox" name="provaBox" >
           <property name="title" >
            <string>Strategia Prove</string>
           </property>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLabel" name="primaLabel" >
                  <property name="text" >
                   <string>Prima Prova:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="secondaLabel" >
                  <property name="text" >
                   <string>Seconda Prova:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="terzaLabel" >
                  <property name="text" >
                   <string>Terza Prova:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="quartaLabel" >
                  <property name="text" >
                   <string>Quarta Prova:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="generaleLabel" >
                  <property name="text" >
                   <string>Prova Generale:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="acciaLabel" >
                  <property name="text" >
                   <string>Provaccia:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QComboBox" name="primaCombo" />
                </item>
                <item>
                 <widget class="QComboBox" name="secondaCombo" />
                </item>
                <item>
                 <widget class="QComboBox" name="terzaCombo" />
                </item>
                <item>
                 <widget class="QComboBox" name="quartaCombo" />
                </item>
                <item>
                 <widget class="QComboBox" name="generaleCombo" />
                </item>
                <item>
                 <widget class="QComboBox" name="acciaCombo" />
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="cavalloLabel" >
           <property name="text" >
            <string>textLabel1</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="formaBar" >
           <property name="value" >
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancella</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >barbero.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>StrategiaProve</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>StrategiaProve</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
